serverless/test/forwarder.spec.ts

```diff
@@ -137,8 +137,8 @@ describe("addCloudWatchForwarderSubscriptions", () => {
     });
     expect(resources).toEqual(mockResources([lambda])); // template should not be modified
   });
-
-  it("does not overwrite existing unknown subscription on log group", async () => {
+  //At the moment this test assumes a log group subscriptions is full when it has 2 existing subscriptions.
+  it("does not overwrite existing subscriptions on a log group whose subscriptions slots are full", async () => {
```

Comment:
I would rephrase `a log group whose subscriptions slots are full` -> `a log group that already has the maximum number of subscriptions`. I know it's helpful to think about "slots" or "being full" here but those aren't really terms AWS uses.